bindings/kitsune_client/src/op_store.rs

```diff
@@ -338,6 +353,31 @@ impl OpStore for WtOpStore {
         })
     }
 
+    /// Get the earliest op timestamp in the given arc.
```

Comment:
Here and above, generally don't copy trait method documentation to implementations of the trait because it's just more places to update the same information. It's easy enough to navigate on docs.rs anyway. I'd only add doc comments to an implementation of a trait method if it does something that needs mentioning that isn't covered by the trait documentation.

Reply:
[fixed](85dca7894b9a48407b87a78e656f5070346b5aea)